modgui.injector.js

```diff
@@ -1,168 +1,214 @@
-globalThis.modapi_guikit = `// ModAPI GUI made by TheIdiotPlays
-// https://github.com/TheIdiotPlays
-(() => {
+globalThis.modapi_guikit = `(() => {
   var splashes = [
     "Now with A.I.D.S (automatically injected dedicated server)",
     "Only causes death 90% of the time!",
     "HTML is better.",
-    "https://github.com/EaglerForge",
     "hey you should check out https://github.com/ZXMushroom63/scratch-gui",
     "99% of people stop gambling before they win big.",
     "Now with free estradiol!",
-    "Now with H.I.V (Hyper Injected Virtual-debugger)"
+    "Now with H.I.V (Hyper Injected Virtual-debugger)",
+    "You like eaglers, don't you ≽^•⩊•^≼",
+    "9/11 victims are the fastest readers! They can go through 110 stories in a few seconds.",
```

Comment:
😭

Reply:
I couldn't resist :3

Reply:
I might ass a few of these ngl https://github.com/STPv22/WildSplashTexts/blob/main/splashes.txt